src/elements/fields/PaymentMethodField.jsx

```diff
@@ -0,0 +1,193 @@
+import React, { memo, useEffect } from 'react';
+import {
+  CardElement,
+  Elements,
+  useElements,
+  useStripe
+} from '@stripe/react-stripe-js';
+
+import { getStripePromise } from '../../integrations/stripe';
+import { filterStyleValues } from '../../utils/styles';
+import { filterKeys } from '../../utils/utils';
+
+import InlineTooltip from '../components/Tooltip';
+
+// In order for the stripe card element to operate and show the hybrid fields, stripe must be loaded and 
+// initialized with a key.  In a runtime form, it is crucial to use the real key.  However. in 
+// the dashboard any key will do since the field is not operational nor would we want it to be.
+const stripeKey = 'any-key-does-not-matter-what';
+const stripePromise = getStripePromise();
+// Limit stripe css props to only those that are supported in order to avoid easy to miss warnings
+const supportedStripeCssProps = [
+  'backgroundColor',
+  'color',
+  'fontFamily',
+  'fontSize',
+  'fontSmoothing',
+  'fontStyle',
+  'fontVariant',
+  'fontWeight',
+  'iconColor',
+  'lineHeight',
+  'letterSpacing',
+  'textAlign',
+  'padding',
+  'textDecoration',
+  'textShadow',
+  'textTransform'
+];
+
+const CardField = ({
+  element,
+  runtime,
+  applyStyles,
+  fieldLabel,
+  elementProps = {},
+  setCardElement = () => {},
+  setFieldError = () => {},
+  onChange = () => {},
+  editable = false,
+  inlineError,
+  errorDisplayMode
+}) => {
+  const stripe = useStripe();
+  const elements = useElements();
+
+  useEffect(() => {
+    if (!stripe || !elements) {
+      // Stripe.js has not yet loaded.
+      return;
+    }
+    setCardElement(element.servar.key, elements.getElement(CardElement));
+  }, [stripe, element.servar.key, elements]);
+
+  const cardElementOptions = {
+    style: {
+      base: {
+        textDecoration: 'none', // Bug in card element - force the reset to none
+        ...filterKeys(applyStyles.getTarget('field'), supportedStripeCssProps),
```

Comment:
does the stripe card element support mobile styles? in this current usage of applystyles we are not fetching the mobile styles, which are stored as an object in a media query

Reply:
@bo-dun-1 So I looked into this and understand mobile styles and how they work a bit more.  In general, mobile styles are supported for the payment method field.  However, we do this using media queries.  The Stripe card element does not support this so the mobile styles that are set in media queries directly in the card element's style object are (safely) ignored.  This means that mobile styles like color and the various font styles do not work (mobile settings will not be applied).  However, other mobile styles for things like background color and borders are set in the parent of the card element and DO WORK.

I am sure we could make all the mobile styles work using javascript matchMedia, etc. instead of media queries.  However, I am inclined to ignore this issue for now and I have put a comment similar to the above in the payment method component code.  If you think this must instead be addressed/fixed now, let me know.

Reply:
makes sense! let's ignore for now